doc/license.md

```diff
@@ -0,0 +1,18 @@
+# License
+
+The Data Observatory is a collection of various sources of data with varying licenses. We have worked hard to find you data that will work for the broades set of use-cases. Please still review the below terms for any dataset you use and please respect the rights of the owners of each dataset you use. We list below the third-party datasets that are used in the Data Observatory, along with the terms governing the use of each.
```

Comment:
broadest yes, "links to" is what i meant by terms, fixed

On Mon, May 16, 2016 at 2:59 PM, csobier <notifications@github.com> wrote:

> In doc/license.md
> <https://github.com/CartoDB/observatory-extension/pull/78#discussion_r63406108>
> :
>
> > @@ -0,0 +1,18 @@
> > +# License
> > +
> > +The Data Observatory is a collection of various sources of data with varying licenses. We have worked hard to find you data that will work for the broades set of use-cases. Please still review the below terms for any dataset you use and please respect the rights of the owners of each dataset you use. We list below the third-party datasets that are used in the Data Observatory, along with the terms governing the use of each.
>
> Did you mean "broadest"? Are the "below terms" still coming? Or are you
> just listing the third-party datasets?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/CartoDB/observatory-extension/pull/78/files/5d9a12dac0daedba7ae45a18dc8ba0f508ef66da#r63406108>
>



-- 

andrewxhill <http://twitter.com/andrewxhill>
http://cartodb.com